documents-services/src/main/java/org/exoplatform/documents/rest/DocumentFileRest.java

```diff
@@ -27,6 +27,8 @@
 import javax.annotation.security.RolesAllowed;
 import javax.jcr.AccessDeniedException;
 import jakarta.servlet.http.HttpServletRequest;
+
+import javax.swing.text.DocumentFilter;
```

Comment:
I believe this is  a wrong import of class ?

Reply:
done